examples/flow.cpp

```diff
@@ -166,8 +167,13 @@ int main(int argc, char** argv)
         }
         // Polymer case
         else if ( phases.active( Opm::Phase::POLYMER ) ) {
-            Opm::flowEbosPolymerSetDeck(*deck, *eclipseState, *schedule, *summary_config);
-            return Opm::flowEbosPolymerMain(argc, argv);
+            if ( phases.size() == 3 ) {
```

Comment:
this part is not for temporary, more careful checking should be made here later.